.rubocop.yml

```diff
@@ -1,26 +1,673 @@
-inherit_mode:
-  merge:
-    - Exclude
-
 require:
-  - standard
-  - standard-custom
-  - standard-performance
-  - standard-rails
-  - rubocop-capybara
-  - rubocop-factory_bot
   - rubocop-performance
   - rubocop-rails
   - rubocop-rspec
+  - rubocop-capybara
+  - rubocop-factory_bot
+
+# Don't even think about it.
+# inherit_from: .rubocop_todo.yml
```

Comment:
can you say something about why?   This isn't terribly helpful.  Also ... does it need to be first in the file?

Reply:
Removed.